ts/features/idpay/initiative/details/components/TimelineTransactionCards.tsx

```diff
@@ -107,3 +132,22 @@ export const IbanOnboardingCard = ({
     </View>
   </BaseTransactionCard>
 );
+
+export const pickTransactionCard = (transaction: OperationListDTO) => {
+  switch (transaction.operationType) {
+    case TransactionOperationTypeEnum.TRANSACTION:
+      return <TimelineTransactionCard transaction={transaction} />;
+    case TransactionOperationTypeEnum.REVERSAL:
+      return <TimelineTransactionCard transaction={transaction} />;
+    case OnboardingOperationTypeEnum.ONBOARDING:
+      return <OnboardingTransactionCard transaction={transaction} />;
+    case InstrumentOperationTypeEnum.ADD_INSTRUMENT:
+      return <InstrumentOnboardingCard transaction={transaction} />;
+    case InstrumentOperationTypeEnum.DELETE_INSTRUMENT:
+      return <InstrumentOnboardingCard transaction={transaction} />;
+    case IbanOperationTypeEnum.ADD_IBAN:
+      return <IbanOnboardingCard transaction={transaction} />;
+    default:
+      return <Text>Error loading {transaction.operationType}</Text>;
+  }
+};
```

Comment:
Why not a single component with parameters instead of a map of components? Could make things way more maintenable.